source/ethereum-clients/cpp-ethereum/automation.rst

```diff
@@ -87,3 +84,51 @@ some key scripts which our automation uses:
 * `scripts/ethbuild.sh <https://github.com/ethereum/webthree-helpers/blob/develop/scripts/ethbuild.sh>`_ - Build code (all platforms)
 * `scripts/ethtests.sh <https://github.com/ethereum/webthree-helpers/blob/develop/scripts/ethtests.sh>`_ - Run tests (all platforms)
 * `scripts/ppabuild.sh <https://github.com/ethereum/webthree-helpers/blob/develop/scripts/ppabuild.sh>`_ - Build bundle for PPAs
+
+But we still have some scripts which are orphaned within Jenkins:
+
+* `Create ZIP of Eth in Windows powershell <http://52.28.164.97/configfiles/show?id=org.jenkinsci.plugins.managedscripts.PowerShellConfig1444842304838>`_ - Used to make win_eth.zip
+* `github_issue_mover.py <http://52.28.164.97/configfiles/show?id=org.jenkinsci.plugins.managedscripts.ScriptConfig1443614334556>`_ - Script used to match move issues from cpp-ethereum to webthree-umbrella repo
+
+
+Setting up a new Jenkins slave
+-------------------------------------------------------------------------------
+
+This is a nightmare process.   Here's how to add an OS X slave.   The process
+for other platforms will vary, but we've not had to do it yet.
+
+- Install appropriate operating system (Bob had to use his own Apple login)
+- Install the latest xcode from the Mac Store
+- Install Homebrew
+    - say yes to xcode select license
+    - brew update
+    - brew upgrade
+    - install pre-requisites (http://www.ethdocs.org/en/latest/ethereum-clients/cpp-ethereum/building-from-source/osx.html)
+    - install Ruby
+        - See https://github.com/rbenv/rbenv#homebrew-on-mac-os-x
+        - brew install rbenv
+        - rbenv init
+        - rbenv install 1.9.3-p551
+        - Add `eval "$(rbenv init -)"` to ~/.bash_profile:
+    - Connect the slave using Java web-start (have to lower security settings)
```

Comment:
That could be changed to a setting where the server logs in via ssh, by the way.